inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -376,6 +378,11 @@ void GNADeviceHelper::close() {
     GNADeviceClose(nGNAHandle);
     nGNAHandle = 0;
 #else
+    if (wasAnyRequestScheduled) {
+        try {
+            wait(lastSheduledRequestId);
+        } catch (...) {}
```

Comment:
at least I would log something, and not hide every exception completelly

Reply:
done